libs/akita/src/lib/query.ts

```diff
@@ -1,22 +1,29 @@
-import { Store } from './store';
 import { Observable } from 'rxjs';
-import { queryConfigKey, QueryConfigOptions } from './queryConfig';
-import { isString } from './isString';
-import { isFunction } from './isFunction';
-import { isDev } from './env';
-import { __queries__ } from './stores';
 import { distinctUntilChanged, map } from 'rxjs/operators';
 import { compareKeys } from './compareKeys';
+import { isDev } from './env';
+import { isFunction } from './isFunction';
+import { isString } from './isString';
+import { queryConfigKey, QueryConfigOptions } from './queryConfig';
+import { isBrowser } from './root';
+import { Store } from './store';
 import { ReturnTypes } from './types';
 
+/** @internal */
```

Comment:
Why it's here?

Reply:
I had to move it here to fix a cyclic import